packages/theming/atlas/scripts/build.js

```diff
@@ -69,8 +69,9 @@ async function buildAndCopyAtlas(watchMode, destination) {
                 killOthers: ["failure"]
             }
         );
-        console.log("Success", success);
-    } catch (failure) {
-        throw new Error(`Failure ${failure}`);
+        console.log("Building & copying Atlas has completed successfully");
+    } catch (commands) {
+        const commandInfo = commands.map(command => `{ name: ${command.command.name}, exit code: ${command.exitCode}}`);
+        throw new Error(`One or more commands failed: ${commandInfo.join(", ")}`);
```

Comment:
Maybe it's easier to read when you `join("\n")`.

Reply:
Much easier indeed. Changed it.